src/types/video.ts

```diff
@@ -350,3 +350,9 @@ export interface ReactVideoProps extends ReactVideoEvents, ViewProps {
   allowsExternalPlayback?: boolean; // iOS
   controlsStyles?: ControlsStyles; // Android
 }
+
+export interface WebStyleProp {
+  style?: CSSProperties;
+}
+
+export type WebReactVideoProps = Omit<ReactVideoProps, 'style'> & WebStyleProp;
```

Comment:
why don't you patch directly ReactVideoProps instead of creating WebStyleProp in this common file ?

Reply:
something like:

```JS
type CustomViewProps = Omit<ViewProps, 'style'> & {
  style?: ViewProps['style'] | CSSProperties;
};

export interface ReactVideoProps extends ReactVideoEvents, CustomViewProps {
```